.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,48 @@
+# this build is designed to replicate the Travis CI workflow
+name: Build with Node
+
+on:
+  push:
+    branches: [ $default-branch, 'github-actions' ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 13.x ]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm ci
+      - run: npm run build --if-present
+      - run: npm test
+
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
FYI @ibacher - I don't actually know if this is enough to do the Coveralls correctly.   I was following https://github.com/marketplace/actions/coveralls-github-action and https://github.com/nickmerwin/node-coveralls and it was unclear to me whether I needed the "finish" step or not.  When I tried adding the finish step in it failed, so I removed it, but not sure that means that things are not actually finishing.  I don't really think Coveralls support should be a dealbreaker anyway - I'm fine getting rid of it personally, especially since this entire repo is deprecated.